render/detailed_node_test.go

```diff
@@ -12,26 +12,30 @@ func TestMakeDetailedNode(t *testing.T) {
 }
 
 func TestOriginTable(t *testing.T) {
-	if _, ok := report.OriginTable(reportFixture, "not-found"); ok {
+	if _, ok := OriginTable(rpt, "not-found"); ok {
 		t.Errorf("unknown origin ID gave unexpected success")
 	}
-	for originID, want := range map[string]report.Table{
-		client54001EndpointNodeID: {
+	for originID, want := range map[string]Table{
+		client54001: {
```

Comment:
client54001 is never an origin node ID, the keys here must take the form xxxNodeID.